File: app/src/main/java/ru/application/homemedkit/connectionController/models/MainModel.java
package ru.application.homemedkit.connectionController.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class MainModel {
    @JsonProperty("codeFounded")
    public boolean codeFounded;
    @JsonProperty("checkResult")
    public boolean checkResult;
    @JsonProperty("category")
    public String category;
    @JsonProperty("cis")
    public String cis;
    @JsonProperty("drugsData")
    public DrugsData drugsData;
}

File: app/src/main/java/ru/application/homemedkit/connectionController/models/MainModel.kt
package ru.application.homemedkit.connectionController.models

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import ru.application.homemedkit.helpers.BLANK

@JsonClass(generateAdapter = true)
data class MainModel(
    @Json val codeFounded: Boolean,
    @Json val checkResult: Boolean,
    @Json val category: String? = BLANK,
    @Json val cis: String = BLANK,
    @Json val drugsData: DrugsData = DrugsData()
)
